projects/packages/stats-admin/src/class-odyssey-config-data.php

```diff
@@ -158,13 +158,13 @@ protected function get_admin_path() {
 	/**
 	 * Get locale acceptable by Calypso.
 	 */
-	protected function get_site_locale() {
+	protected function get_user_locale() {
 		/**
 		 * In WP, locales are formatted as LANGUAGE_REGION, for example `en`, `en_US`, `es_AR`,
 		 * but Calypso expects language-region, e.g. `en-us`, `en`,  `es-ar`. So we need to convert
 		 * them to lower case and replace the underscore with a dash.
 		 */
-		$locale = strtolower( get_locale() );
+		$locale = strtolower( get_user_locale() );
```

Comment:
Im a bit confused. At first glance this seems like it would be an infinite loop?

Reply:
No, because we're calling the `get_user_locale` global function from the `Odyssey_Config_Data::get_user_locale` method, so they are two different functions.